app/domain/SignTx.js

```diff
@@ -21,23 +24,23 @@ export type TrezorSignTxPayload = {
 
 export type BIP32Path = Array<number>;
 
-export type InputTypeUTxO = {|
+export type LedgerInputTypeUTxO = {
```

Comment:
@sarveshwar-singh I don’t understand why the file was changed to a generic name and this types were changed from generic to Ledger related 🤔

Reply:
@nicarq 
Changing to generic because it store properties for Trezor and Ledger payload types.

 InputTypeUTxO -> LedgerInputTypeUTxO (to make it understandable that it belongs to ledger)

NOTE: 
1. For Ledger: this is temporary  (see the comment at top of the file), when ADA related `nmp` is out then these will no longer need.

2. For Trezor: these could not be exported from Trezor provided library ?

Reply:
`InputTypeUTxO -> LedgerInputTypeUTxO (to make it understandable that it belongs to ledger)`

This was already used by Trezor

Reply:
@nicarq 
Sorry i did not understand your point here.